scripts/ldap/setup_ldap.sh

```diff
@@ -87,6 +87,7 @@ rm -f /etc/openldap/slapd.d/cn\=config/cn\=schema/*ppolicy.ldif
 # Removing configuration files
```

Comment:
Please add a comment mentioning that we need to delete all prev loaded modules else we may see opendap idempotency issue